website/staticsite.yaml

```diff
@@ -1,27 +1,38 @@
 baseurl: ''
 events:
 - badges:
-    - href: 'https://calendar.google.com/event?action=TEMPLATE&tmeid=MDZoZzAyZTRiOTB2OHZrZDQyNGFpa2pwY3MgcHlqYWlwdXIuaW5kaWFAbQ&tmsrc=pyjaipur.india%40gmail.com'
-      text: Add to calendar
-    - href: 'https://meet.google.com/ecf-eubt-bor'
-      text: Call
-    - href: 'https://github.com/PyJaipur/PyJaipur/issues/149'
-      text: Github
+  - href: 'https://calendar.google.com/event?action=TEMPLATE&tmeid=NWk1NXEzdjg2Z2dlZWVzNGMwYTZrM3IxazUgcHlqYWlwdXIuaW5kaWFAbQ&tmsrc=pyjaipur.india%40gmail.com'
+    text: Add to calendar
+  - href: 'https://meet.google.com/tqc-yjap-aaw'
+    text: Call
+  - href: https://github.com/PyJaipur/PyJaipur/issues/153
+    text: Github
+  desc: We're having a community call this sunday at 9 pm! Come join the community!
+  start: 13 September 2020 21:00:00 GMT+5:30
+  title: Community call
```

Comment:
@C-mmon we can change this title and description to match whatever is needed for the 13th talk.